### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by bittu4** [reggie3/react-native-webview-braintree#8]
Added webview Braintree using npm 
when used in the expo project getting below error
ERROR
10:48
Building JavaScript bundle: error
ERROR
10:48
Unable to resolve "AccessibilityInfo" from "node_modules/react-native-webview-braintree/node_modules/react-native/Libraries/react-native/react-native-implementation.js"

Can anyone please tell me what's happening and please tell me how to fix this
